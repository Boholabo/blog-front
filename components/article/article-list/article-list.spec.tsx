import React from 'react';
import { render } from '../../../test/testUtils';
import ArticleList from './index';

jest.mock('next/router', () => ({
  useRouter() {
    return {
      route: '/',
      pathname: '',
      query: '',
      asPath: '',
    };
  },
}));

describe('article/list component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <ArticleList article={{ data: { data: [] } }} />,
      {}
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
