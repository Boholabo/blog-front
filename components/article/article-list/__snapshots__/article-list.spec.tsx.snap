// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`article/list component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="article-list-header"
    >
      <div
        class="list-header"
      >
        <div
          class="logo-box"
        >
          <p
            class="logo"
          >
            logo
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
